stdlib/public/core/SwiftifyImport.swift

```diff
@@ -34,6 +39,9 @@ public enum _SwiftifyInfo {
     /// object past the lifetime of the function.
     /// Parameter pointer: index of pointer in function parameter list.
     case nonescaping(pointer: _SwiftifyExpr)
+    /// Can express lifetime dependencies between inputs and outputs of a function.
+    /// 'from' is the input on which the output 'to' depends.
+    case lifetimeDependence(from: Int, to: _SwiftifyExpr, type: DependenceType)
```

Comment:
How will borrow vs copy be indicated from C++?

Reply:
It depends on whether the lifetimebound is applied to an escapable or a non-escapable type. It is borrow in the former case and copy in the latter.